daemon/api.go

```diff
@@ -86,37 +86,42 @@ var (
 	}
 
 	snapsCmd = &Command{
-		Path:   "/2.0/snaps",
-		UserOK: true,
-		GET:    getSnapsInfo,
-		POST:   sideloadSnap,
+		Path:            "/2.0/snaps",
+		UserOK:          true,
+		GET:             getSnapsInfo,
+		POST:            sideloadSnap,
+		PolicyKitAction: "com.ubuntu.snappy.sideload-snap",
```

Comment:
Personally I'd prefer to have a few documented constants that are referenced from all the Command objects.